prow/config.yaml

```diff
@@ -2083,6 +2237,8 @@ presubmits:
     # TODO(bentheelder): enforce this pattern (skip all releases) or implement
     # something better like https://github.com/kubernetes/test-infra/pull/4918
     skip_branches:
+    - release-1.10 # per-release image
```

Comment:
nit: this is reverse order from the other jobs

Reply:
I should flip the other ones, this is the most common existing order because it mirrors the order of the job entries below it.

Reply:
I've normalized all skip_branches to be in descending order